composer.json

```diff
@@ -43,7 +43,7 @@
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<2.1 || >=3.0",
-        "jms/serializer": "<0.13 || >=2.0",
```

Comment:
This should be the right way, because we only test version 1.0 - 3.9999

```
"jms/serializer": "<1.0 || >=3.0",
```

Reply:
done